mllib/src/main/scala/org/apache/spark/mllib/clustering/LDAOptimizer.scala

```diff
@@ -497,40 +495,38 @@ final class OnlineLDAOptimizer extends LDAOptimizer with Logging {
       (u._1, u._2, u._3 + v._3)
     }
 
-    val (statsSum: BDM[Double], logphatOption: Option[BDV[Double]], nonEmptyDocsN: Long) = stats
+    val (statsSum: BDM[Double], logphatOption: Option[BDV[Double]], batchSize: Long) = stats
       .treeAggregate((BDM.zeros[Double](k, vocabSize), logphatPartOptionBase(), 0L))(
         elementWiseSum, elementWiseSum
       )
```

Comment:
Not a big deal, but I think here use `mapPartition`(do aggregate in it) and then do treeAggregate, the process can be merged into one `treeAggregate`, is it right? it make the code more clear.

Reply:
Yes, that's right. But on the other hand, we've currently got a rather simple `treeAggregate` so it barely contributes to the complexity of the code. 

Anyway, is it OK to fix this in this issue?

Reply:
hmm... I think it only need to move the code block in `mapPartition` into the aggregate `seqOp` function. Will it make code more complicated ?
I think it can be fix in this PR, it is only a simple modification and do not affect the logic.

Reply:
Actually, the block from `mapPartition` will be simplified, since it will no longer need to process collections of documents.

Reply:
It looks better now. Thank you for the suggestion.